server/routerlicious/packages/routerlicious-base/src/utils/constants.ts

```diff
@@ -7,4 +7,6 @@ export const Constants = Object.freeze({
 	alfredRestThrottleIdSuffix: "AlfredRest",
 	createDocThrottleIdPrefix: "createDoc",
 	getDeltasThrottleIdPrefix: "getDeltas",
+    getSessionThrottleIdPrefix: "getSession",
+    generalRestCallThrottleIdPrefix: "generalRestCall",
```

Comment:
nit: same here, tabs/spacing off. Please run `pnpm run prettier:fix` in server/routerlicious thanks!

Reply:
Done. :)